packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -199,11 +206,39 @@ const startBatch = async function (
     const cacheAllScenarios = batchTrRoutingConfig.cacheAllScenarios;
     const debugFromParamOrConfig = debug !== undefined ? debug : batchTrRoutingConfig.debug;
 
+    // Start Memcached
+    const startMemcached = true; // TODO implement a config option to allow disable this
```

Comment:
allow to disable, but also configure this (host/port)

Reply:
Yes, there's many thing that could be configured.